src/languages/en.ts

```diff
@@ -1735,33 +1737,33 @@ export default {
         hasBeenThrottledError: 'An error occurred while adding your bank account. Please wait a few minutes and try again.',
         hasCurrencyError: 'Oops! It appears that your workspace currency is set to a different currency than USD. To proceed, please set it to USD and try again.',
         error: {
-            youNeedToSelectAnOption: 'Please select an option to proceed',
-            noBankAccountAvailable: "Sorry, there's no bank account available",
-            noBankAccountSelected: 'Please choose an account',
-            taxID: 'Please enter a valid tax ID number',
-            website: 'Please enter a valid website using lower-case letters',
+            youNeedToSelectAnOption: 'Please select an option to proceed.',
+            noBankAccountAvailable: "Sorry, there's no bank account available.",
+            noBankAccountSelected: 'Please choose an account.',
+            taxID: 'Please enter a valid tax ID number.',
+            website: 'Please enter a valid website using lower-case letters.',
```

Comment:
this came from `rulesdir/use-periods-for-error-messages` auto-fix, not sure if this should apply to translations tho